python/revng/internal/cli/_commands/test_docs/doctest_runner.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+
+import argparse
+import doctest
+import importlib.util
+import sys
+
+NO_DIFF = doctest.register_optionflag("NO_DIFF")
```

Comment:
Nice thing.

1. I'd call the feature "ignoreoutput".
2. I see sometimes this is used to ignore *all* the outputs of a snippet. Can we have make this happen automatically if you don't specify any line?
   Alternatively, we could have `ignorealloutput`.

Reply:
1. Done
2. Done, if you do `{python ignoreoutput=1,2}` it will ignore lines 1 and 2, whereas if you do `{python ignoreoutput}` it will ignore everything